src/lines/LinesCanvas.js

```diff
@@ -10,21 +10,139 @@ import {
 
+
 export const LinesCanvas = () => {
   const ref = useRef(null);
   const replayBtn = useRef(null);
   //
+
+
+
   useEffect(() => {
-    let mini = new Mini({ name: "base", domElement: ref.current, window });
 
+    // load GLTF loader 
+    const loadModel = async (mini,uri) => 
+    {
+    return new Promise(resolve => {
+      var loaderModel = new GLTFLoader();
+      const dracoLoader = new DRACOLoader();
+      dracoLoader.setDecoderConfig( { type: 'jsm' } );
+      //dracoLoader.setDecoderPath( "three/examples/js/libs/draco/" );
+      dracoLoader.setDecoderPath( "http://192.168.0.22:5000/draco/" );
+      loaderModel.setDRACOLoader( dracoLoader );
+
+      loaderModel.load(
+        uri,
+        model => {
+
+          let geometries = []
+          let objects_name = []
+
+          // find all geometries in model
+          model.scene.traverse( function ( object ) {
+            if ( object.isMesh === true ) {
+              geometries.push( object );
+            }
+          });
+
+          let placement = new Vector3(-10.0, +9.0, -0.0) ; //new Vector3(+60.0, -40.0, -5.0) ;
+
+          geometries.forEach( object => {
+            let object_name = object.name;
+            let object_geo = object.geometry;
+            object_geo.scale(5,5,5);
+
+            new LineStuff(mini, {
+              name: object_name,
+              baseGeometry: object_geo,
+              position: placement,
+            });
+            objects_name.push(object_name)
+          } )
+
+          console.error(objects_name)
+          let play = () => {
+
+            let mini_objects = []
+
+            objects_name.forEach(element => {
+              mini_objects.push(mini.i[element])
+            });
+
+            Promise.all([
+              mini.i.floor,
+              mini.i.item1,
+              mini.i.item2,
+              mini.i.item3,
+              //
+            ].concat(mini_objects)).then( (array_items) => {
+              
+              array_items.forEach( (item) => {
+                item.hide();
+              })
+      
+              array_items.forEach( (item, idx) => {
+                item.run({ delay: idx*30 });
+              })
+
+            });
+          };
+          play();
+          replayBtn.current.style.cursor = "pointer";
+          replayBtn.current.addEventListener("click", () => {
+            play();
+          });
+          resolve(model);
+        }
+      );
+    });
+    };
+
+    //const uris = ["http://localhost:5000/2018_Mclaren_senna.gltf"]
+    //const uris = ["http://localhost:5000/showroom_garments.gltf"]
+    //const uris = ["http://localhost:5000/showroom_motorcycles.gltf"]
+    //const uris = ["http://localhost:5000/villa_joli.gltf"]
+    //const uris = ["http://localhost:5000/house_joli.gltf"]
+    //const uris = ["http://localhost:5000/DE-LEB-08.gltf"]
+    //const uris = ["http://localhost:5000/DE-LEB-08.gltf"]
+    const uris = ["http://192.168.0.22:5000/katedra-processed.gltf"]
```

Comment:
https://vercel.com/wonglok831/loklok-volume/3bafhF5AjZtbfhBxLvKr7WbZqimL

here's a preview link of the brach,